roles/app/tasks/docker.yml

```diff
@@ -61,6 +71,8 @@
       ACCESS_TOKEN_ENC_SECRET: "{{ access_token_secret }}"
       AMQP_URL: "{{ amqp_protocol }}://{{ rabbitmq_username }}:{{ rabbitmq_password }}@{{ rabbitmq_host }}/{{ rabbitmq_vhost | urlencode | replace('/', '%2F') }}"
       DATABASE_URL: "ecto://{{ wanda_postgres_user }}:{{ wanda_postgres_password }}@{{ wanda_postgres_host }}/{{ wanda_postgres_db }}"
+    volumes_from:
```

Comment:
Maybe it is worth to upgrade from volumes_from to just volumes as it is deprecated?

Reply:
Let me look better into that, I am not sure anymore it even works 🙈

Reply:
Hmm maybe i was wrong as it is deprecated for docker-compose but i guess we are fine in docker:
https://docs.docker.com/reference/cli/docker/container/run/#volumes-from

Reply:
Alright it is actually working 
- https://github.com/trento-project/web/pull/3456 
- https://3456.prenv.trento.suse.com/